modules/rfg-icon.yml

```diff
@@ -6,7 +6,7 @@ icon: real-favicon-generator.png
 github: 'https://github.com/pimlie/nuxt-rfg-icon'
 website: 'https://github.com/pimlie/nuxt-rfg-icon'
 learn_more: 'https://realfavicongenerator.net/'
-category: Devtools
+category: Extensions
```

Comment:
Shall we move this to `UI` since it is related to visual changes? cc @pi0

Reply:
SEO may also make sense (since PWA is there too)

Reply:
Agree!